templates/layouts/_base.html

```diff
@@ -56,8 +56,10 @@
   {% if cookie_settings_url %}
     {{
       onsCookiesBanner({
-          "html": _('We use cookies to improve your experience of census.gov.uk'),
-          "secondaryButtonUrl": cookie_settings_url
+        "secondaryButtonUrl": cookie_settings_url,
+        "statementTitle": _('Tell us whether you accept cookies'),
+        "statementText": _("We use <a href='{cookie_settings_url}'>cookies to collect information</a> about how you use census.gov.uk. We use this information to make the website work as well as possible and improve our services.").format(cookie_settings_url=cookie_settings_url),
+        "confirmationText": _("You’ve accepted all cookies. You can <a href='{cookie_settings_url}'>change your cookie preferences</a> at any time.").format(cookie_settings_url=cookie_settings_url),
```

Comment:
I think `primaryButtonText`, `secondaryButtonText` and `confirmationButtonText` need to be passed to the macro so that translations will work.